src/Components/Wizard/StepMarker/index.tsx

```diff
@@ -1,112 +1,60 @@
-import Icon from "../Icon"
+import Icon from "../../Icon"
 import React, { Component } from "react"
-import Text from "../Text"
-import colors from "../../Assets/Colors"
+import Text from "../../Text"
+import colors from "../../../Assets/Colors"
 import styled from "styled-components"
-import { Fonts } from "../Publishing/Fonts"
-import { Props, State, StepProps } from "./types"
-import { isUndefined } from "lodash"
+import { Fonts } from "../../Publishing/Fonts"
+import { Props, State, StepState } from "./types"
+
+export * from "./types"
 
 export class StepMarker extends Component<Props, State> {
+  static defaultProps = {
+    disableInternalState: false,
+  }
+
```

Comment:
I took @damassi 's Stepmarker, moved it and simplified it to a point that it probably could be an SFC now, but this was the last piece of what I did before submitting the PR and thought I shouldn't undo too much of what was already there. I did move it inside of the `Wizard`.